test/run-test.rb

```diff
@@ -13,6 +13,7 @@
   require "minitest"
   require "rr"
 when "minitest-active-support"
+  require "active_support"
```

Comment:
Could you show the error message without this?

Reply:
Here you go.  I've run into this several times now when adding Rails 7 support to gems.  

Rails repo discussion is here - https://github.com/rails/rails/issues/43851
My fork error is here - https://github.com/petergoldstein/rr/runs/5472214118?check_suite_focus=true

Screenshot:
<img width="780" alt="Screen Shot 2022-03-08 at 3 35 22 PM" src="https://user-images.githubusercontent.com/421488/157343923-aa209b6a-f10f-43f6-8ef8-e36f78c94eb4.png">

Reply:
Thanks.
Can we remove `require "active_support/test_case"`?

Reply:
@kou Nope, it breaks specs.

Reply:
https://github.com/petergoldstein/rr/actions/runs/1954535292

Reply:
OK.